test/unit/openmrs.spec.js

```diff
@@ -0,0 +1,147 @@
+    });
+
+    beforeEach(module('conceptDictionaryApp'));
+
+    describe('ListFull should retrun full representations', function(){
+        var $httpBackend, openmrsRest, list;
+
+        beforeEach(inject(function(_$httpBackend_, _openmrsRest_){
+        	openmrsRest = _openmrsRest_;
+            $httpBackend = _$httpBackend_;
+            $httpBackend.whenGET('partials/index-menu.html').respond();
+            $httpBackend.whenGET('/ws/rest/v1/item?v=full').
+            respond({results:[{ name: 'item1',  
+            					uuid: "2b22dc27-72ec-4ab5-9fa8-d98be91adc1c"},
+            				  { name: 'item2', 
+            					uuid: "3g65dc27-72ec-4ab5-9fa8-d98be91adb5b"},
+            					{ name: 'item3', 
+                				uuid: "6g65dc56-72ec-4ab5-9fa8-d98be91adb5b"}
+            				]});      
+            
+            openmrsRest.listFull('item').then(function(resp){
+            	list = resp;
+            });
+            
+            $httpBackend.flush();
+        }));
+        
+        it('should load items list', function(){
+        	expect(list.results).toEqualData([{ name: 'item1',  
+				            					uuid: "2b22dc27-72ec-4ab5-9fa8-d98be91adc1c"},
+				            				  { name: 'item2', 
+				            					uuid: "3g65dc27-72ec-4ab5-9fa8-d98be91adb5b"},
+				            					{ name: 'item3', 
+				                				uuid: "6g65dc56-72ec-4ab5-9fa8-d98be91adb5b"}]
+				            				);
+        });
+    });
+    describe('Get should retrun one item', function(){
+        var $httpBackend, openmrsRest, item;
+
+        beforeEach(inject(function(_$httpBackend_, _openmrsRest_){
+        	openmrsRest = _openmrsRest_;
+            $httpBackend = _$httpBackend_;
+            $httpBackend.whenGET('partials/index-menu.html').respond();
+            $httpBackend.whenGET('/ws/rest/v1/item/6g65dc56-72ec-4ab5-9fa8-d98be91adb5b').
+            respond({ name: 'item3', 
+                	uuid: "6g65dc56-72ec-4ab5-9fa8-d98be91adb5b"});
+            
+            openmrsRest.get('item', {uuid: "6g65dc56-72ec-4ab5-9fa8-d98be91adb5b"}).then(function(resp){
+            	item = resp;
+            });
+            
+            $httpBackend.flush();
+        }));
+        it('should load item3', function(){
+        	expect(item).toEqualData({ name: 'item3', 
+        								uuid: "6g65dc56-72ec-4ab5-9fa8-d98be91adb5b"});	
+        });
+    });
+    describe('Create method should create new item', function(){
+    	var $httpBackend, openmrsRest, POSTitem;
+
+        beforeEach(inject(function(_$httpBackend_, _openmrsRest_){
+	    	openmrsRest = _openmrsRest_;
+	        $httpBackend = _$httpBackend_;
+	        $httpBackend.whenGET('partials/index-menu.html').respond();
+	        $httpBackend.whenPOST('/ws/rest/v1/item').
+	        respond({ name: 'item4', 
+	            	uuid: "6g65dc56-72ec-4ab5-9fa8-d98be91adb5b"});
+	        
+	        var newItem = angular.toJson({name:'item4',uuid: "6g65dc56-72ec-4ab5-9fa8-d98be91adb5b"});
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Mixed double and single quotes.](https://www.codacy.com/app/openmrs/openmrs-owa-conceptdictionary/file/2319554381/issues/source?bid=3196969&fileBranchId=3204313#l90)